### ISSUE TITLE
UnsupportedEncodingException: "utf-8"

### ISSUE THREAD

**Issue by lopotun** [softwaremill/sttp#467]
When I try to reach Facebook HP at http://www.facebook.com/ I get UnsupportedEncodingException: "utf-8" exception:
```
Caused by: java.io.UnsupportedEncodingException: "utf-8"
	at java.lang.StringCoding.decode(StringCoding.java:190)
	at java.lang.String.<init>(String.java:426)
	at java.lang.String.<init>(String.java:491)
	at sttp.client.SttpApi.$anonfun$asStringAlways$1(SttpApi.scala:77)
	at sttp.client.MappedResponseAs.$anonfun$mapWithMetadata$1(ResponseAs.scala:44)
	at sttp.client.HttpURLConnectionBackend.readResponseBody(HttpURLConnectionBackend.scala:251)
...

```
Test code:
```
  implicit val backend = HttpURLConnectionBackend()
  def test() = {
      val url = "http://www.facebook.com/"
      import sttp.client._
      val request = basicRequest.head(uri"$url")
      val response = request.send()
      response.code
    }
```
Indeed, FB sends `content-type: text/html; charset="utf-8"` in its response (with redundant "s). Should the extra " be removed in method `sttp.client.SttpApi#asStringAlways`?

**Comment by lopotun** [comment]
Superfast! Thanks!

**Comment by adamw** [comment]
Released in 2.0.5
